src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import "./inputWithLabel.css"
+function InputWithLabel({ Labelvalue, placeholder, multiline }) {
+    if (!multiline) {
+        return (
+            <div className="input">
+                <label className="label">{Labelvalue}</label>
+                <input type="text" placeholder={placeholder}/>
+            </div>
+        );
+    } else {
+        return (
+            <div className="input">
+                <label className="label">{Labelvalue}</label>
+                <textarea placeholder={placeholder} />
+            </div>
+        );
+    }
+}
```

Comment:
There is code duplication here always stick to this principle `Don't repeat yourself`
Here is a better way to write this 
```js
	return (
		<div className="input">
			<label className="label">{Labelvalue}</label>
			{multiline ? (
				<textarea placeholder={placeholder} />
			) : (
				<input type="text" placeholder={placeholder} />
			)}
		</div>
	);
```